### ISSUE TITLE
Implement content audit view of banner image on the healthcare system

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#13672]
##### Background
There are some editors using a specific image type to market information, this is incorrect use of that image type and alongside the removal of existing full width banners we'd like to provide specific guidance on where these images are being incorrectly used AND offer guidance in KB articles about banners to not use these images like this instead 

The banner image on the healthcare system is decorative and has been hardcoded with a null alt. If editors upload a non-decorative image into this field this will create an inaccessible experience for the image

##### Scope questions:
- what is the name of the field housing the images we want to audit? The banner image on the healthcare system 

##### Acceptance Criteria 
- [ ] I expect a lasting view is created that shows the healthcare system and the associated banner image for the healthcare system
- [ ] I expect the view contains the following fields : `system name`, `image (same aspect ratio as the front end)`  
- [ ] I expect the view is styled according to guidelines 
- [ ] I expect the view lives in content audits: https://prod.cms.va.gov/admin/content/audit
- [ ] @kmariepat-cityfriends to review report and share results with VHA DM (this AC can be marked as complete once report is complete and added as agenda item to reoccurring VHA DM sync)

Next steps on another card:
Discuss with VHA DM 
- [ ] I expect the banner image on the healthcare system should be locked down so that they can only be maintained via request fulfillment

**Comment by omahane** [comment]
@kmariepat-cityfriends The PR is up. Given that it's an audit for you, it might be good for your to review it. https://github.com/department-of-veterans-affairs/va.gov-cms/pull/13988
